assets/variant-list.js

```diff
@@ -133,7 +133,7 @@ class VariantList extends HTMLElement {
         selector: '.shopify-section'
```

Comment:
Buttons jump to the side on mobile: https://screenshot.click/30-53-hew2h-r4xmq.mp4